src/pudl/transform/params/ferc1.py

```diff
@@ -2992,7 +2992,7 @@
         "drop_invalid_rows": [
             {
                 "invalid_values": [pd.NA, np.nan, ""],
-                "required_valid_cols": ["energy_source_mwh"],
+                "required_valid_cols": ["energy_mwh"],
```

Comment:
This naming inconsistency resulted in all of the data being dropped, and we were not catching it. We need to be adding some sanity checks to the data validations so at least the nightly builds catch this kind of thing.

Reply:
i noticed this in one of my recent table transforms. think `drop_invalid_rows` should also just let you know if a `required_valid_cols` isn't in the table at all

Reply:
There was a warning in the logging output, but it didn't fail.

Reply:
More generally I think we should be adding data validation expectations to these tables as we add them to the database -- the null columns check would have found this, as would a validation indicating the expected number of rows.

Reply:
i 100% agree with you here and also we have so many tables to integrate in the next few weeks. I would like to be able to add data validations in a thoughtful way and I don't think we actually have time for that right now.